app/assets/stylesheets/tasks.scss

```diff
@@ -14,6 +24,35 @@
     }
   }
 
+  @keyframes thumbwobble {
+    0% {-webkit-transform: rotate(0deg);
+      opacity: 1;
+      font-size: 6em;
+    }
+    50% {-webkit-transform: rotate(30deg);
+      font-size: 9em;
+      opacity: 1;
+    }
+    90% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 0;
+    }
+
+  }
+  .thumbsup {
```

Comment:
A space between `}` and `.thumbsup` would be nice! :smile:

Reply:
done :)

Reply:
Sorry to be picky. :grimacing: But I meant it the other way around. 
```css
}
# ← here
.thumbsup {
```

I could have been more clear :D

Reply:
That's actually what I did ... or do I (still) understand you wrong?

Reply:
Yip, it lookes like the space is between the `}` rather than between the sections.
So instead of 

```css
  }

}
.thumbsup {
```

this would look better:

```css
  }
}

.thumbsup {
```